src/db_iter.h

```diff
@@ -98,14 +98,14 @@ class TitanDBIterator : public Iterator {
   }
 
   Slice value() const override {
-    assert(Valid());
+    assert(Valid() && !options_.key_only);
```

Comment:
return `Slice()` if `key_only=true`?

Reply:
should we abort or set the status to `not ok`?

Reply:
Just return `Slice()`? I think it's ok to call `value()` in case `key_only=true`.